website/pages/en/sunrise.mdx

```diff
@@ -152,7 +152,11 @@ No, all infrastructure is operated by independent Indexers on The Graph Network,
 
 You can use the [Subgraph Studio](https://thegraph.com/studio/) to create, test, and publish your subgraph. All hosted service users must upgrade their subgraph to The Graph Network before the end of the upgrade window ending on June 12th 2024.
 
-The upgrade Indexer ensures you can query your subgraph even without curation signal.
+The [Sunrise Upgrade Indexer](/sunrise/#what-is-the-upgrade-indexer) ensures the indexing of all subgraphs. However, signaling GRT on a particular subgraph will draw more indexers to it. This incentivization of additional Indexers through curation aims to enhance the quality of service for queries by reducing latency and enhancing network availability.
+
+When signaling, Curators can decide to signal on a specific version of the subgraph or to signal using auto-migrate. If they signal using auto-migrate, a curator’s shares will always be updated to the latest version published by the developer. If they decide to signal on a specific version instead, shares will always stay on that specific version.
+
+To assist teams that are transitioning subgraphs from the hosted service to the Graph Network, curation support is now available. If you require assistance with curation to enhance the quality of service, please send a request to the Edge & Node team at support@thegraph.zendesk.com and specify the subgraphs you need assistance with.
```

Comment:
Is it intentional that these 3 paragraphs are found verbatim on two pages? I'm slightly worried that if we make changes to one, we forget to make changes to the other one. In fact, the first paragraph is slightly different: the first sentence from above is split in two here, with "However" at the start of the first sentence – I much prefer it like that!

Reply:
This was Adam's suggestion, but I had the same exact concern that you did.  TY, I was a fan of the "however" myself!

Reply:
I think we can link people to a single source of the copy if we prefer, but we should link to that from several places. I think "Publishing a subgraph" is probably the right source of truth

Reply:
@azf20 For now, will copy in multiple places, but will update in the future. Are we still okay leaving the email visible?